// cowsay/build/cowsay.es.js
import ackbar_cow from '.cows/ackbar.cow';
import apertureBlank_cow from '.cows/aperture-blank.cow';
import aperture_cow from '.cows/aperture.cow';
import armadillo_cow from '.cows/armadillo.cow';
import atat_cow from '.cows/atat.cow';
import atom_cow from '.cows/atom.cow';
import awesomeFace_cow from '.cows/awesome-face.cow';
import banana_cow from '.cows/banana.cow';
import bearface_cow from '.cows/bearface.cow';
import bees_cow from '.cows/bees.cow';
import billTheCat_cow from '.cows/bill-the-cat.cow';
import biohazard_cow from '.cows/biohazard.cow';
import bishop_cow from '.cows/bishop.cow';
import blackMesa_cow from '.cows/black-mesa.cow';
import box_cow from '.cows/box.cow';
import brokenHeart_cow from '.cows/broken-heart.cow';
import C3PO_cow from '.cows/C3PO.cow';
import cake_cow from '.cows/cake.cow';
import cakeWithCandles_cow from '.cows/cake-with-candles.cow';
import cat2_cow from '.cows/cat2.cow';
import cat_cow from '.cows/cat.cow';
import catfence_cow from '.cows/catfence.cow';
import charizardvice_cow from '.cows/charizardvice.cow';
import charlie_cow from '.cows/charlie.cow';
import chessmen_cow from '.cows/chessmen.cow';
import chito_cow from '.cows/chito.cow';
import clawArm_cow from '.cows/claw-arm.cow';
import clippy_cow from '.cows/clippy.cow';
import companionCube_cow from '.cows/companion-cube.cow';
import cowfee_cow from '.cows/cowfee.cow';
import cthulhuMini_cow from '.cows/cthulhu-mini.cow';
import cube_cow from '.cows/cube.cow';
import dalek_cow from '.cows/dalek.cow';
import dalekShooting_cow from '.cows/dalek-shooting.cow';
import dockerWhale_cow from '.cows/docker-whale.cow';
import dolphin_cow from '.cows/dolphin.cow';
import ebi_furai_cow from '.cows/ebi_furai.cow';
import elephant2_cow from '.cows/elephant2.cow';
import elephant_cow from '.cows/elephant.cow';
import explosion_cow from '.cows/explosion.cow';
import fatBanana_cow from '.cows/fat-banana.cow';
import fatCow_cow from '.cows/fat-cow.cow';
import fence_cow from '.cows/fence.cow';
import fire_cow from '.cows/fire.cow';
import fox_cow from '.cows/fox.cow';
import ghost_cow from '.cows/ghost.cow';
import glados_cow from '.cows/glados.cow';
import goat2_cow from '.cows/goat2.cow';
import goldenEagle_cow from '.cows/golden-eagle.cow';
import hand_cow from '.cows/hand.cow';
import happyWhale_cow from '.cows/happy-whale.cow';
import hippie_cow from '.cows/hippie.cow';
import hiya_cow from '.cows/hiya.cow';
import hiyoko_cow from '.cows/hiyoko.cow';
import homer_cow from '.cows/homer.cow';
import hypno_cow from '.cows/hypno.cow';
import ibm_cow from '.cows/ibm.cow';
import iwashi_cow from '.cows/iwashi.cow';
import jellyfish_cow from '.cows/jellyfish.cow';
import karl_marx_cow from '.cows/karl_marx.cow';
import kilroy_cow from '.cows/kilroy.cow';
import king_cow from '.cows/king.cow';
import kitten_cow from '.cows/kitten.cow';
import knight_cow from '.cows/knight.cow';
import lamb2_cow from '.cows/lamb2.cow';
import lamb_cow from '.cows/lamb.cow';
import lightbulb_cow from '.cows/lightbulb.cow';
import lobster_cow from '.cows/lobster.cow';
import lollerskates_cow from '.cows/lollerskates.cow';
import mailchimp_cow from '.cows/mailchimp.cow';
import mazeRunner_cow from '.cows/maze-runner.cow';
import minotaur_cow from '.cows/minotaur.cow';
import monaLisa_cow from '.cows/mona-lisa.cow';
import mooghidjirah_cow from '.cows/mooghidjirah.cow';
import moojira_cow from '.cows/moojira.cow';
import mule_cow from '.cows/mule.cow';
import nyan_cow from '.cows/nyan.cow';
import octopus_cow from '.cows/octopus.cow';
import okazu_cow from '.cows/okazu.cow';
import owl_cow from '.cows/owl.cow';
import pawn_cow from '.cows/pawn.cow';
import periodicTable_cow from '.cows/periodic-table.cow';
import personalitySphere_cow from '.cows/personality-sphere.cow';
import pinballMachine_cow from '.cows/pinball-machine.cow';
import psychiatrichelp2_cow from '.cows/psychiatrichelp2.cow';
import psychiatrichelp_cow from '.cows/psychiatrichelp.cow';
import pterodactyl_cow from '.cows/pterodactyl.cow';
import queen_cow from '.cows/queen.cow';
import R2D2_cow from '.cows/R2-D2.cow';
import radio_cow from '.cows/radio.cow';
import renge_cow from '.cows/renge.cow';
import robot_cow from '.cows/robot.cow';
import robotfindskitten_cow from '.cows/robotfindskitten.cow';
import roflcopter_cow from '.cows/roflcopter.cow';
import rook_cow from '.cows/rook.cow';
import sachiko_cow from '.cows/sachiko.cow';
import seahorseBig_cow from '.cows/seahorse-big.cow';
import seahorse_cow from '.cows/seahorse.cow';
import shikato_cow from '.cows/shikato.cow';
import shrug_cow from '.cows/shrug.cow';
import smilingOctopus_cow from '.cows/smiling-octopus.cow';
import snoopy_cow from '.cows/snoopy.cow';
import snoopyhouse_cow from '.cows/snoopyhouse.cow';
import snoopysleep_cow from '.cows/snoopysleep.cow';
import spidercow_cow from '.cows/spidercow.cow';
import squid_cow from '.cows/squid.cow';
import sudowoodo_cow from '.cows/sudowoodo.cow';
import tableflip_cow from '.cows/tableflip.cow';
import taxi_cow from '.cows/taxi.cow';
import template_cow from '.cows/template.cow';
import threader_cow from '.cows/threader.cow';
import threecubes_cow from '.cows/threecubes.cow';
import toaster_cow from '.cows/toaster.cow';
import tortoise_cow from '.cows/tortoise.cow';
import tuxBig_cow from '.cows/tux-big.cow';
import tweetyBird_cow from '.cows/tweety-bird.cow';
import USA_cow from '.cows/USA.cow';
import vader_cow from '.cows/vader.cow';
import weepingAngel_cow from '.cows/weeping-angel.cow';
import wizard_cow from '.cows/wizard.cow';
import wood_cow from '.cows/wood.cow';
import world_cow from '.cows/world.cow';
import yasuna_01_cow from '.cows/yasuna_01.cow';
import yasuna_02_cow from '.cows/yasuna_02.cow';
import yasuna_03a_cow from '.cows/yasuna_03a.cow';
import yasuna_03_cow from '.cows/yasuna_03.cow';
import yasuna_04_cow from '.cows/yasuna_04.cow';
import yasuna_05_cow from '.cows/yasuna_05.cow';
import yasuna_06_cow from '.cows/yasuna_06.cow';
import yasuna_07_cow from '.cows/yasuna_07.cow';
import yasuna_08_cow from '.cows/yasuna_08.cow';
import yasuna_09_cow from '.cows/yasuna_09.cow';
import yasuna_10_cow from '.cows/yasuna_10.cow';
import yasuna_11_cow from '.cows/yasuna_11.cow';
import yasuna_12_cow from '.cows/yasuna_12.cow';
import yasuna_13_cow from '.cows/yasuna_13.cow';
import yasuna_14_cow from '.cows/yasuna_14.cow';
import yasuna_16_cow from '.cows/yasuna_16.cow';
import yasuna_17_cow from '.cows/yasuna_17.cow';
import yasuna_18_cow from '.cows/yasuna_18.cow';
import yasuna_19_cow from '.cows/yasuna_19.cow';
import yasuna_20_cow from '.cows/yasuna_20.cow';
import ymd_udon_cow from '.cows/ymd_udon.cow';
import zenNohMilk_cow from '.cows/zen-noh-milk.cow';

function createCommonjsModule(fn, module) {
	return module = { exports: {} }, fn(module, module.exports), module.exports;
}

var ansiRegex = createCommonjsModule(function (module) {
'use strict';

module.exports = () => {
	const pattern = [
		'[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\\u0007)',
		'(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))'
	].join('|');

	return new RegExp(pattern, 'g');
};
});

var stripAnsi = input => typeof input === 'string' ? input.replace(ansiRegex(), '') : input;

var isFullwidthCodePoint = createCommonjsModule(function (module) {
'use strict';
/* eslint-disable yoda */
module.exports = x => {
	if (Number.isNaN(x)) {
		return false;
	}

	// code points are derived from:
	// http://www.unix.org/Public/UNIDATA/EastAsianWidth.txt
	if (
		x >= 0x1100 && (
			x <= 0x115f ||  // Hangul Jamo
			x === 0x2329 || // LEFT-POINTING ANGLE BRACKET
			x === 0x232a || // RIGHT-POINTING ANGLE BRACKET
			// CJK Radicals Supplement .. Enclosed CJK Letters and Months
			(0x2e80 <= x && x <= 0x3247 && x !== 0x303f) ||
			// Enclosed CJK Letters and Months .. CJK Unified Ideographs Extension A
			(0x3250 <= x && x <= 0x4dbf) ||
			// CJK Unified Ideographs .. Yi Radicals
			(0x4e00 <= x && x <= 0xa4c6) ||
			// Hangul Jamo Extended-A
			(0xa960 <= x && x <= 0xa97c) ||
			// Hangul Syllables
			(0xac00 <= x && x <= 0xd7a3) ||
			// CJK Compatibility Ideographs
			(0xf900 <= x && x <= 0xfaff) ||
			// Vertical Forms
			(0xfe10 <= x && x <= 0xfe19) ||
			// CJK Compatibility Forms .. Small Form Variants
			(0xfe30 <= x && x <= 0xfe6b) ||
			// Halfwidth and Fullwidth Forms
			(0xff01 <= x && x <= 0xff60) ||
			(0xffe0 <= x && x <= 0xffe6) ||
			// Kana Supplement
			(0x1b000 <= x && x <= 0x1b001) ||
			// Enclosed Ideographic Supplement
			(0x1f200 <= x && x <= 0x1f251) ||
			// CJK Unified Ideographs Extension B .. Tertiary Ideographic Plane
			(0x20000 <= x && x <= 0x3fffd)
		)
	) {
		return true;
	}

	return false;
};
});

var stringWidth = createCommonjsModule(function (module) {
'use strict';



module.exports = str => {
	if (typeof str !== 'string' || str.length === 0) {
		return 0;
	}

	str = stripAnsi(str);

	let width = 0;

	for (let i = 0; i < str.length; i++) {
		const code = str.codePointAt(i);

		// Ignore control characters
		if (code <= 0x1F || (code >= 0x7F && code <= 0x9F)) {
			continue;
		}

		// Ignore combining characters
		if (code >= 0x300 && code <= 0x36F) {
			continue;
		}

		// Surrogates
		if (code > 0xFFFF) {
			i++;
		}

		width += isFullwidthCodePoint(code) ? 2 : 1;
	}

	return width;
};
});

var say$1 = function (text, wrap) {
	var delimiters = {
		first : ["/", "\\"],
		middle : ["|", "|"],
		last : ["\\", "/"],
		only : ["<", ">"]
	};

	return format(text, wrap, delimiters);
};

var think$1 = function (text, wrap) {
	var delimiters = {
		first : ["(", ")"],
		middle : ["(", ")"],
		last : ["(", ")"],
		only : ["(", ")"]
	};

	return format(text, wrap, delimiters);
};

function format (text, wrap, delimiters) {
	var lines = split(text, wrap);
	var maxLength = max(lines);

	var balloon;
	if (lines.length === 1) {
		balloon = [
			" " + top(maxLength),
			delimiters.only[0] + " " + lines[0] + " " + delimiters.only[1],
			" " + bottom(maxLength)
		];
	} else {
		balloon = [" " + top(maxLength)];

		for (var i = 0, len = lines.length; i < len; i += 1) {
			var delimiter;

			if (i === 0) {
				delimiter = delimiters.first;
			} else if (i === len - 1) {
				delimiter = delimiters.last;
			} else {
				delimiter = delimiters.middle;
			}

			balloon.push(delimiter[0] + " " + pad(lines[i], maxLength) + " " + delimiter[1]);
		}

		balloon.push(" " + bottom(maxLength));
	}

	return balloon.join("\n");
}

function split (text, wrap) {
	text = text.replace(/\r\n?|[\n\u2028\u2029]/g, "\n").replace(/^\uFEFF/, '').replace(/\t/g, '        ');

	var lines = [];
	if (!wrap) {
		lines = text.split("\n");
	} else {
		var start = 0;
		while (start < text.length) {
			var nextNewLine = text.indexOf("\n", start);

			var wrapAt = Math.min(start + wrap, nextNewLine === -1 ? text.length : nextNewLine);

			lines.push(text.substring(start, wrapAt));
			start = wrapAt;

			// Ignore next new line
			if (text.charAt(start) === "\n") {
				start += 1;
			}
		}
	}

	return lines;
}

function max (lines) {
	var max = 0;
	for (var i = 0, len = lines.length; i < len; i += 1) {
		if (stringWidth(lines[i]) > max) {
			max = stringWidth(lines[i]);
		}
	}

	return max;
}

function pad (text, length) {
	return text + (new Array(length - stringWidth(text) + 1)).join(" ");
}

function top (length) {
	return new Array(length + 3).join("_");
}

function bottom (length) {
	return new Array(length + 3).join("-");
}

var balloon = {
	say: say$1,
	think: think$1
};

var replacer = function (cow, variables) {
	var eyes = escapeRe(variables.eyes);
	var eyeL = eyes.charAt(0);
	var eyeR = eyes.charAt(1);
	var tongue = escapeRe(variables.tongue);

	if (cow.indexOf("$the_cow") !== -1) {
		cow = extractTheCow(cow);
	}

	return cow
		.replace(/\$thoughts/g, variables.thoughts)
		.replace(/\$eyes/g, eyes)
		.replace(/\$tongue/g, tongue)
		.replace(/\$\{eyes\}/g, eyes)
		.replace(/\$eye/, eyeL)
		.replace(/\$eye/, eyeR)
		.replace(/\$\{tongue\}/g, tongue)
	;
};

/*
 * "$" dollar signs must be doubled before being used in a regex replace
 * This can occur in eyes or tongue.
 * For example:
 *
 * cowsay -g Moo!
 *
 * cowsay -e "\$\$" Moo!
 */
function escapeRe (s) {
	if (s && s.replace) {
		return s.replace(/\$/g, "$$$$");
	}
	return s;
}

function extractTheCow (cow) {
	cow = cow.replace(/\r\n?|[\n\u2028\u2029]/g, "\n").replace(/^\uFEFF/, '');
	var match = /\$the_cow\s*=\s*<<"*EOC"*;*\n([\s\S]+)\nEOC\n/.exec(cow);

	if (!match) {
		console.error("Cannot parse cow file\n", cow);
		return cow;
	} else {
		return match[1].replace(/\\{2}/g, "\\").replace(/\\@/g, "@").replace(/\\\$/g, "$");
	}
}

var modes = {
	"b" : {
		eyes : "==",
		tongue : "  "
	},
	"d" : {
		eyes : "xx",
		tongue : "U "
	},
	"g" : {
		eyes : "$$",
		tongue : "  "
	},
	"p" : {
		eyes : "@@",
		tongue : "  "
	},
	"s" : {
		eyes : "**",
		tongue : "U "
	},
	"t" : {
		eyes : "--",
		tongue : "  "
	},
	"w" : {
		eyes : "OO",
		tongue : "  "
	},
	"y" : {
		eyes : "..",
		tongue : "  "
	}
};

var faces = function (options) {
	for (var mode in modes) {
		if (options[mode] === true) {
			return modes[mode];
		}
	}

	return {
		eyes : options.e || "oo",
		tongue : options.T || "  "
	};
};

var DEFAULT_COW = "$the_cow = <<\"EOC\";\n        $thoughts   ^__^\n         $thoughts  ($eyes)\\\\_______\n            (__)\\\\       )\\\\/\\\\\n             $tongue ||----w |\n                ||     ||\nEOC\n";

var beavis_zen = "##\n## Beavis, with Zen philosophy removed.\n##\n$the_cow = <<EOC;\n   $thoughts         __------~~-,\n    $thoughts      ,'            ,\n          /               \\\\\n         /                :\n        |                  '\n        |                  |\n        |                  |\n         |   _--           |\n         _| =-.     .-.   ||\n         $eye|/$eye/       _.   |\n         /  ~          \\\\ |\n       (____\\@)  ___~    |\n          |_===~~~.`    |\n       _______.--~     |\n       \\\\________       |\n                \\\\      |\n              __/-___-- -__\n             /            _ \\\\\nEOC\n";

var bong = "##\n## A cow with a bong, from lars@csua.berkeley.edu\n##\n$the_cow = <<EOC;\n         $thoughts\n          $thoughts\n            ^__^ \n    _______/($eyes)\n/\\\\/(       /(__)\n   | W----|| |~|\n   ||     || |~|  ~~\n             |~|  ~\n             |_| o\n             |#|/\n            _+#+_\nEOC\n";

var budFrogs = "##\n## The Budweiser frogs\n##\n$the_cow = <<EOC;\n     $thoughts\n      $thoughts\n          oO)-.                       .-(Oo\n         /__  _\\\\                     /_  __\\\\\n         \\\\  \\\\(  |     ()~()         |  )/  /\n          \\\\__|\\\\ |    (-___-)        | /|__/\n          '  '--'    ==`-'==        '--'  '\nEOC\n";

var bunny = "##\n## A cute little wabbit\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts   \\\\\n        \\\\ /\\\\\n        ( )\n      .( o ).\nEOC\n";

var cheese = "##\n## The cheese from milk & cheese\n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts\n      _____   _________\n     /     \\\\_/         |\n    |                 ||\n    |                 ||\n   |    ###\\\\  /###   | |\n   |     $eye  \\\\/  $eye    | |\n  /|                 | |\n / |        <        |\\\\ \\\\\n| /|                 | | |\n| |     \\\\_______/   |  | |\n| |        $tongue       | / /\n/||                 /|||\n   ----------------|\n        | |    | |\n        ***    ***\n       /___\\\\  /___\\\\\nEOC\n";

var cower = "##\n## A cowering cow\n##\n$the_cow = <<EOC;\n     $thoughts\n      $thoughts\n        ,__, |    | \n        ($eyes)\\\\|    |___\n        (__)\\\\|    |   )\\\\_\n         $tongue  |    |_w |  \\\\\n             |    |  ||   *\n\n             Cower....\nEOC\n";

var daemon = "##\n## 4.4 >> 5.4\n##\n$the_cow = <<EOC;\n   $thoughts         ,        ,\n    $thoughts       /(        )`\n     $thoughts      \\\\ \\\\___   / |\n            /- _  `-/  '\n           (/\\\\/ \\\\ \\\\   /\\\\\n           / /   | `    \\\\\n           $eye $eye   ) /    |\n           `-^--'`<     '\n          (_.)  _  )   /\n           `.___/`    /\n             `-----' /\n<----.     __ / __   \\\\\n<----|====O)))==) \\\\) /====\n<----'    `--' `.__,' \\\\\n             |        |\n              \\\\       /\n        ______( (_  / \\\\______\n      ,'  ,-----'   |        \\\\\n      `--{__________)        \\\\/\nEOC\n";

var doge = "##\n## Doge\n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts\n\n           _                _\n          / /.           _-//\n         / ///         _-   /\n        //_-//=========     /\n      _///        //_ ||   ./\n    _|                 -__-||\n   |  __              - \\\\   \\\n  |  |#-       _-|_           |\n  |            |#|||       _   |  \n |  _==_                       ||\n- ==|.=.=|_ =                  |\n|  |-|-  ___                  |\n|    --__   _                /\n||     ===                  |\n |                     _. //\n  ||_         __-   _-  _|\n     \\_______/  ___/  _|\n                   --*\nEOC\n";

var dragonAndCow = "##\n## A dragon smiting a cow, possible credit to kube@csua.berkeley.edu\n##\n$the_cow = <<EOC;\n                       $thoughts                    ^    /^\n                        $thoughts                  / \\\\  // \\\\\n                         $thoughts   |\\\\___/|      /   \\\\//  .\\\\\n                          $thoughts  /O  O  \\\\__  /    //  | \\\\ \\\\           *----*\n                            /     /  \\\\/_/    //   |  \\\\  \\\\          \\\\   |\n                            \\@___\\@`    \\\\/_   //    |   \\\\   \\\\         \\\\/\\\\ \\\\\n                           0/0/|       \\\\/_ //     |    \\\\    \\\\         \\\\  \\\\\n                       0/0/0/0/|        \\\\///      |     \\\\     \\\\       |  |\n                    0/0/0/0/0/_|_ /   (  //       |      \\\\     _\\\\     |  /\n                 0/0/0/0/0/0/`/,_ _ _/  ) ; -.    |    _ _\\\\.-~       /   /\n                             ,-}        _      *-.|.-~-.           .~    ~\n            \\\\     \\\\__/        `/\\\\      /                 ~-. _ .-~      /\n             \\\\____($eyes)           *.   }            {                   /\n             (    (--)          .----~-.\\\\        \\\\-`                 .~\n             //__\\\\\\\\$tongue\\\\__ Ack!   ///.----..<        \\\\             _ -~\n            //    \\\\\\\\               ///-._ _ _ _ _ _ _{^ - - - - ~\nEOC\n";

var dragon = "##\n## The Whitespace Dragon\n##\n$the_cow = <<EOC;\n      $thoughts                    / \\\\  //\\\\\n       $thoughts    |\\\\___/|      /   \\\\//  \\\\\\\\\n            /$eye  $eye  \\\\__  /    //  | \\\\ \\\\    \n           /     /  \\\\/_/    //   |  \\\\  \\\\  \n           \\@_^_\\@'/   \\\\/_   //    |   \\\\   \\\\ \n           //_^_/     \\\\/_ //     |    \\\\    \\\\\n        ( //) |        \\\\///      |     \\\\     \\\\\n      ( / /) _|_ /   )  //       |      \\\\     _\\\\\n    ( // /) '/,_ _ _/  ( ; -.    |    _ _\\\\.-~        .-~~~^-.\n  (( / / )) ,-{        _      `-.|.-~-.           .~         `.\n (( // / ))  '/\\\\      /                 ~-. _ .-~      .-~^-.  \\\\\n (( /// ))      `.   {            }                   /      \\\\  \\\\\n  (( / ))     .----~-.\\\\        \\\\-'                 .~         \\\\  `. \\\\^-.\n             ///.----..>        \\\\             _ -~             `.  ^-`  ^-_\n               ///-._ _ _ _ _ _ _}^ - - - - ~                     ~-- ,.-~\n                                                                  /.-~\nEOC\n";

var elephantInSnake = "##\n## Do we need to explain this?\n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts              ....       \n           ........    .      \n          .            .      \n         .             .      \n.........              .......\n..............................\n\nElephant inside ASCII snake\nEOC\n";

var eyes = "##\n## Evil-looking eyes\n##\n$the_cow = <<EOC;\n    $thoughts\n     $thoughts\n                                   .::!!!!!!!:.\n  .!!!!!:.                        .:!!!!!!!!!!!!\n  ~~~~!!!!!!.                 .:!!!!!!!!!UWWW\\$\\$\\$ \n      :\\$\\$NWX!!:           .:!!!!!!XUWW\\$\\$\\$\\$\\$\\$\\$\\$\\$P \n      \\$\\$\\$\\$\\$##WX!:      .<!!!!UW\\$\\$\\$\\$\"  \\$\\$\\$\\$\\$\\$\\$\\$# \n      \\$\\$\\$\\$\\$  \\$\\$\\$UX   :!!UW\\$\\$\\$\\$\\$\\$\\$\\$\\$   4\\$\\$\\$\\$\\$* \n      ^\\$\\$\\$B  \\$\\$\\$\\$\\\\     \\$\\$\\$\\$\\$\\$\\$\\$\\$\\$\\$\\$   d\\$\\$R\" \n        \"*\\$bd\\$\\$\\$\\$      '*\\$\\$\\$\\$\\$\\$\\$\\$\\$\\$\\$o+#\" \n             \"\"\"\"          \"\"\"\"\"\"\" \nEOC\n";

var flamingSheep = "##\n## The flaming sheep, contributed by Geordan Rosario (geordan@csua.berkeley.edu)\n##\n$the_cow = <<EOC;\n  $thoughts            .    .     .   \n   $thoughts      .  . .     `  ,     \n    $thoughts    .; .  : .' :  :  : . \n     $thoughts   i..`: i` i.i.,i  i . \n      $thoughts   `,--.|i |i|ii|ii|i: \n           U${eyes}U\\\\.'\\@\\@\\@\\@\\@\\@`.||' \n           \\\\__/(\\@\\@\\@\\@\\@\\@\\@\\@\\@\\@)'  \n             $tongue (\\@\\@\\@\\@\\@\\@\\@\\@)    \n                `YY~~~~YY'    \n                 ||    ||     \nEOC\n";

var ghostbusters = "##\n## Ghostbusters!\n##\n$the_cow = <<EOC;\n          $thoughts\n           $thoughts\n            $thoughts          __---__\n                    _-       /--______\n               __--( /     \\\\ )XXXXXXXXXXX\\\\v.\n             .-XXX(   $eye   $eye  )XXXXXXXXXXXXXXX-\n            /XXX(       U     )        XXXXXXX\\\\\n          /XXXXX(              )--_  XXXXXXXXXXX\\\\\n         /XXXXX/ (      O     )   XXXXXX   \\\\XXXXX\\\\\n         XXXXX/   /            XXXXXX   \\\\__ \\\\XXXXX\n         XXXXXX__/          XXXXXX         \\\\__---->\n ---___  XXX__/          XXXXXX      \\\\__         /\n   \\\\-  --__/   ___/\\\\  XXXXXX            /  ___--/=\n    \\\\-\\\\    ___/    XXXXXX              '--- XXXXXX\n       \\\\-\\\\/XXX\\\\ XXXXXX                      /XXXXX\n         \\\\XXXXXXXXX   \\\\                    /XXXXX/\n          \\\\XXXXXX      >                 _/XXXXX/\n            \\\\XXXXX--__/              __-- XXXX/\n             -XXXXXXXX---------------  XXXXXX-\n                \\\\XXXXXXXXXXXXXXXXXXXXXXXXXX/\n                  \"\"VXXXXXXXXXXXXXXXXXXV\"\"\nEOC\n";

var goat = "##\n## ejm97 http://www.ascii-art.de/ascii/ghi/goat.txt\n##\n$the_cow = <<EOC;\n       $thoughts\n        $thoughts\n         $thoughts  _))\n           > $eye\\\\     _~\n           `;'\\\\\\\\__-' \\\\_\n              | )  _ \\\\ \\\\\n             / / ``   w w\n            w w\nEOC\n\n\n\n\n\n";

var hedgehog = "##\n## A cute little hedgehog\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts ..:::::::::.\n    ::::::::::::::\n   /. `::::::::::::\n  O__,_:::::::::::'\nEOC\n";

var hellokitty = "##\n## Hello Kitty\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts\n      /\\\\_)o<\n     |      \\\\\n     | $eye . $eye|\n      \\\\_____/\n         $tongue\nEOC\n";

var kiss = "##\n## A lovers' empbrace\n##\n$the_cow = <<EOC;\n     $thoughts\n      $thoughts\n             ,;;;;;;;,\n            ;;;;;;;;;;;,\n           ;;;;;'_____;'\n           ;;;(/))))|((\\\\\n           _;;((((((|))))\n          / |_\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\n     .--~(  \\\\ ~))))))))))))\n    /     \\\\  `\\\\-(((((((((((\\\\\\\\\n    |    | `\\\\   ) |\\\\       /|)\n     |    |  `. _/  \\\\_____/ |\n      |    , `\\\\~            /\n       |    \\\\  \\\\           /\n      | `.   `\\\\|          /\n      |   ~-   `\\\\        /\n       \\\\____~._/~ -_,   (\\\\\n        |-----|\\\\   \\\\    ';;\n       |      | :;;;'     \\\\\n      |  /    |            |\n      |       |            |\nEOC\n";

var kitty = "##\n## A kitten of sorts, I think\n##\n$the_cow = <<EOC;\n     $thoughts\n      $thoughts\n       (\"`-'  '-/\") .___..--' ' \"`-._\n         ` $eye_ $eye  )    `-.   (      ) .`-.__. `)\n         (_Y_.) ' ._   )   `._` ;  `` -. .-'\n      _.. `--'_..-_/   /--' _ .' ,4\n   ( i l ),-''  ( l i),'  ( ( ! .-'    \nEOC\n";

var koala = "##\n## From the canonical koala collection\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts\n       ___  \n     {~$eye_$eye~}\n      ( Y )\n     ()~*~()   \n     (_)-(_)   \nEOC\n";

var kosh = "##\n## It's a Kosh Cow!\n##\n$the_cow = <<EOC;\n    $thoughts\n     $thoughts\n      $thoughts\n  ___       _____     ___\n /   \\\\     /    /|   /   \\\\\n|     |   /    / |  |     |\n|     |  /____/  |  |     |     \n|     |  |    |  |  |     |\n|     |  | {} | /   |     |\n|     |  |____|/    |     |\n|     |    |==|     |     |\n|      \\\\___________/      |\n|                         |\n|                         |\nEOC\n";

var lukeKoala = "##\n## From the canonical koala collection\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts          .\n       ___   //\n     {~$eye_$eye~}// \n      ( Y )K/  \n     ()~*~()   \n     (_)-(_)   \n     Luke    \n     Sywalker\n     koala   \nEOC\n";

var mechAndCow = "$the_cow = <<EOC;\n      $thoughts                            |     |\n       $thoughts                        ,--|     |-.\n                         __,----|  |     | |\n                       ,;::     |  `_____' |\n                       `._______|    i^i   |\n                                `----| |---'| .\n                           ,-------._| |== ||//\n                           |       |_|P`.  /'/\n                           `-------' 'Y Y/'/'\n                                     .==\\ /_\\\n   ^__^                             /   /'|  `i\n   ($eyes)\\_______                   /'   /  |   |\n   (__)\\       )\\/\\             /'    /   |   `i\n    $tongue ||----w |           ___,;`----'.___L_,-'`\\__\n       ||     ||          i_____;----\\.____i\"\"\\____\\\nEOC\n";

var meow = "##\n## A meowing tiger?\n##\n$the_cow = <<EOC;\n  $thoughts\n   $thoughts ,   _ ___.--'''`--''//-,-_--_.\n      \\\\`\"' ` || \\\\\\\\ \\\\ \\\\\\\\/ / // / ,-\\\\\\\\`,_\n     /'`  \\\\ \\\\ || Y  | \\\\|/ / // / - |__ `-,\n    /\\$eye\"\\\\  ` \\\\ `\\\\ |  | ||/ // | \\\\/  \\\\  `-._`-,_.,\n   /  _.-. `.-\\\\,___/\\\\ _/|_/_\\\\_\\\\/|_/ |     `-._._)\n   `-'``/  /  |  // \\\\__/\\\\__  /  \\\\__/ \\\\\n    $tongue  `-'  /-\\\\/  | -|   \\\\__ \\\\   |-' |\n          __/\\\\ / _/ \\\\/ __,-'   ) ,' _|'\n         (((__/(((_.' ((___..-'((__,'\nEOC\n";

var milk = "##\n## Milk from Milk and Cheese\n##\n$the_cow = <<EOC;\n $thoughts     ____________ \n  $thoughts    |__________|\n      /           /\\\\\n     /           /  \\\\\n    /___________/___/|\n    |          |     |\n    |  ==\\\\ /== |     |\n    |   $eye   $eye  | \\\\ \\\\ |\n    |     <    |  \\\\ \\\\|\n   /|          |   \\\\ \\\\\n  / |  \\\\_____/ |   / /\n / /|    $tongue    |  / /|\n/||\\\\|          | /||\\\\/\n    -------------|   \n        | |    | | \n       <__/    \\\\__>\nEOC\n";

var moofasa = "##\n## MOOfasa.\n##\n$the_cow = <<EOC;\n       $thoughts    ____\n        $thoughts  /    \\\\\n          | ^__^ |\n          | ($eyes) |______\n          | (__) |      )\\\\/\\\\\n           \\\\____/|----w |\n                ||     ||\n\n\t         Moofasa\nEOC\n";

var moose = "$the_cow = <<EOC;\n  $thoughts\n   $thoughts   \\\\_\\\\_    _/_/\n    $thoughts      \\\\__/\n           ($eyes)\\\\_______\n           (__)\\\\       )\\\\/\\\\\n            $tongue ||----- |\n               ||     ||\nEOC\n";

var mutilated = "##\n## A mutilated cow, from aspolito@csua.berkeley.edu\n##\n$the_cow = <<EOC;\n       $thoughts   \\\\_______\n v__v   $thoughts  \\\\   O   )\n ($eyes)      ||----w |\n (__)      ||     ||  \\\\/\\\\\n  $tongue\nEOC\n";

var ren = "##\n## Ren \n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts\n    ____  \n   /# /_\\\\_\n  |  |/$eye\\\\$eye\\\\\n  |  \\\\\\\\_/_/\n / |_   |  \n|  ||\\\\_ ~| \n|  ||| \\\\/  \n|  |||_    \n \\\\//  |    \n  ||  |    \n  ||_  \\\\   \n  \\\\_|  o|  \n  /\\\\___/   \n /  ||||__ \n    (___)_)\nEOC\n";

var satanic = "##\n## Satanic cow, source unknown.\n##\n$the_cow = <<EOC;\n     $thoughts\n      $thoughts  (__)  \n         (\\\\/)  \n  /-------\\\\/    \n / | 666 ||$tongue  \n*  ||----||      \n   ~~    ~~      \nEOC\n";

var sheep = "##\n## The non-flaming sheep.\n##\n$the_cow = <<EOC\n  $thoughts\n   $thoughts\n       __     \n      U${eyes}U\\\\.'\\@\\@\\@\\@\\@\\@`.\n      \\\\__/(\\@\\@\\@\\@\\@\\@\\@\\@\\@\\@)\n        $tongue (\\@\\@\\@\\@\\@\\@\\@\\@)\n           `YY~~~~YY'\n            ||    ||\nEOC\n";

var skeleton = "##\n## This 'Scowleton' brought to you by one of \n## {appel,kube,rowe}@csua.berkeley.edu\n##\n$the_cow = <<EOC;\n          $thoughts      (__)      \n           $thoughts     /$eyes|  \n            $thoughts   (_\"_)*+++++++++*\n                   //I#\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\I\\\\\n                   I[I|I|||||I I `\n                   I`I'///'' I I\n                   I I       I I\n                   ~ ~       ~ ~\n                     Scowleton\nEOC\n";

var small = "##\n## A small cow, artist unknown\n##\n$eyes = \"..\" unless ($eyes);\n$the_cow = <<EOC;\n       $thoughts   ,__,\n        $thoughts  ($eyes)____\n           (__)    )\\\\\n            $tongue||--|| *\nEOC\n";

var squirrel = "$the_cow = <<EOC;\n  $thoughts\n     $thoughts\n                  _ _\n       | \\__/|  .~    ~.\n       /$eyes `./      .'\n      {o__,   \\    {\n        / .  . )    \\\n        `-` '-' \\    }\n       .(   _(   )_.'\n      '---.~_ _ _|\n                                                     \nEOC\n";

var stegosaurus = "##\n## A stegosaur with a top hat?\n##\n$the_cow = <<EOC;\n$thoughts                             .       .\n $thoughts                           / `.   .' \" \n  $thoughts                  .---.  <    > <    >  .---.\n   $thoughts                 |    \\\\  \\\\ - ~ ~ - /  /    |\n         _____          ..-~             ~-..-~\n        |     |   \\\\~~~\\\\.'                    `./~~~/\n       ---------   \\\\__/                        \\\\__/\n      .'  $eye    \\\\     /               /       \\\\  \" \n     (_____,    `._.'               |         }  \\\\/~~~/\n      `----.          /       }     |        /    \\\\__/\n            `-.      |       /      |       /      `. ,~~|\n                ~-.__|      /_ - ~ ^|      /- _      `..-‘ / \\\\  /\\\\\n                     |     /        |     /     ~-.     `-/ _ \\\\/__\\\\\n                     |_____|        |_____|         ~ - . _ _ _ _ _>\nEOC\n";

var stimpy = "##\n## Stimpy!\n##\n$the_cow = <<EOC;\n  $thoughts     .    _  .    \n   $thoughts    |\\\\_|/__/|    \n       / / \\\\/ \\\\  \\\\  \n      /__|$eye||$eye|__ \\\\ \n     |/_ \\\\_/\\\\_/ _\\\\ |  \n     | | (____) | ||  \n     \\\\/\\\\___/\\\\__/  // \n     (_/         ||\n      |          ||\n      |          ||\\\\   \n       \\\\        //_/  \n        \\\\______//\n       __ || __||\n      (____(____)\nEOC\n";

var supermilker = "##\n## A cow being milked, probably from Lars Smith (lars@csua.berkeley.edu)\n##\n$the_cow = <<EOC;\n  $thoughts   ^__^\n   $thoughts  ($eyes)\\\\_______        ________\n      (__)\\\\       )\\\\/\\\\    |Super |\n       $tongue ||----W |       |Milker|\n          ||    UDDDDDDDDD|______|\nEOC\n";

var surgery = "##\n## A cow operation, artist unknown\n##\n$the_cow = <<EOC;\n          $thoughts           \\\\  / \n           $thoughts           \\\\/  \n               (__)    /\\\\         \n               ($eyes)   O  O        \n               _\\\\/_   //         \n         *    (    ) //       \n          \\\\  (\\\\\\\\    //       \n           \\\\(  \\\\\\\\    )                              \n            (   \\\\\\\\   )   /\\\\                          \n  ___[\\\\______/^^^^^^^\\\\__/) o-)__                     \n |\\\\__[=======______//________)__\\\\                    \n \\\\|_______________//____________|                    \n     |||      || //||     |||\n     |||      || @.||     |||                        \n      ||      \\\\/  .\\\\/      ||                        \n                 . .                                 \n                '.'.`                                \n\n            COW-OPERATION                           \nEOC\n";

var telebears = "##\n## A cow performing an unnatural act, artist unknown.\n##\n$the_cow = <<EOC;\n      $thoughts                _\n       $thoughts              (_)   <-- TeleBEARS\n        $thoughts   ^__^       / \\\\\n         $thoughts  ($eyes)\\\\_____/_\\\\ \\\\\n            (__)\\\\  you  ) /\n             $tongue ||----w ((\n                ||     ||>> \nEOC\n";

var turkey = "##\n## Turkey!\n##\n$the_cow = <<EOC;\n  $thoughts                                  ,+*^^*+___+++_\n   $thoughts                           ,*^^^^              )\n    $thoughts                       _+*                     ^**+_\n     $thoughts                    +^       _ _++*+_+++_,         )\n              _+^^*+_    (     ,+*^ ^          \\\\+_        )\n             {       )  (    ,(    ,_+--+--,      ^)      ^\\\\\n            { (\\@)    } f   ,(  ,+-^ __*_*_  ^^\\\\_   ^\\\\       )\n           {:;-/    (_+*-+^^^^^+*+*<_ _++_)_    )    )      /\n          ( /  (    (        ,___    ^*+_+* )   <    <      \\\\\n           U _/     )    *--<  ) ^\\\\-----++__)   )    )       )\n            (      )  _(^)^^))  )  )\\\\^^^^^))^*+/    /       /\n          (      /  (_))_^)) )  )  ))^^^^^))^^^)__/     +^^\n         (     ,/    (^))^))  )  ) ))^^^^^^^))^^)       _)\n          *+__+*       (_))^)  ) ) ))^^^^^^))^^^^^)____*^\n          \\\\             \\\\_)^)_)) ))^^^^^^^^^^))^^^^)\n           (_             ^\\\\__^^^^^^^^^^^^))^^^^^^^)\n             ^\\\\___            ^\\\\__^^^^^^))^^^^^^^^)\\\\\\\\\n                  ^^^^^\\\\uuu/^^\\\\uuu/^^^^\\\\^\\\\^\\\\^\\\\^\\\\^\\\\^\\\\^\\\\\n                     ___) >____) >___   ^\\\\_\\\\_\\\\_\\\\_\\\\_\\\\_\\\\)\n                    ^^^//\\\\\\\\_^^//\\\\\\\\_^       ^(\\\\_\\\\_\\\\_\\\\)\n                      ^^^ ^^ ^^^ ^\nEOC\n";

var turtle = "##\n## A mysterious turtle...\n##\n$the_cow = <<EOC;\n    $thoughts                                  ___-------___\n     $thoughts                             _-~~             ~~-_\n      $thoughts                         _-~                    /~-_\n             /^\\\\__/^\\\\         /~  \\\\                   /    \\\\\n           /|  $eye|| $eye|        /      \\\\_______________/        \\\\\n          | |___||__|      /       /                \\\\          \\\\\n          |          \\\\    /      /                    \\\\          \\\\\n          |   (_______) /______/                        \\\\_________ \\\\\n          |      $tongue / /         \\\\                      /            \\\\\n           \\\\         \\\\^\\\\\\\\         \\\\                  /               \\\\     /\n             \\\\         ||           \\\\______________/      _-_       //\\\\__//\n               \\\\       ||------_-~~-_ ------------- \\\\ --/~   ~\\\\    || __/\n                 ~-----||====/~     |==================|       |/~~~~~\n                  (_(__/  ./     /                    \\\\_\\\\      \\\\.\n                         (_(___/                         \\\\_____)_)\nEOC\n";

var tux = "##\n## TuX\n## (c) pborys@p-soft.silesia.linux.org.pl \n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts\n        .--.\n       |$eye_$eye |\n       |:_/ |\n      //   \\\\ \\\\\n     (|     | )\n    /'\\\\_   _/`\\\\\n    \\\\___)=(___/\n\nEOC\n";

var vaderKoala = "##\n## Another canonical koala?\n##\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts        .\n     .---.  //\n    Y|$eye $eye|Y// \n   /_(i=i)K/ \n   ~()~*~()~  \n    (_)-(_)   \n\n     Darth \n     Vader    \n     koala        \nEOC\n";

var whale = "# whale\n#\n# modified from https://www.reddit.com/r/pics/comments/25ji0n/man_face_to_face_with_whale/chi1kdy?context=3\n$the_cow = <<EOC;\n   $thoughts\n    $thoughts\n     $thoughts\n                '-.\n      .---._     \\\\ \\.--'\n    /       `-..__)  ,-'\n   |    0           /\n    \\--.__,   .__.,`\n     `-.___'._\\\\_.'\n\nEOC\n";

var www = "##\n## A cow wadvertising the World Wide Web, from lim@csua.berkeley.edu\n##\n$the_cow = <<EOC;\n        $thoughts   ^__^\n         $thoughts  ($eyes)\\\\_______\n            (__)\\\\       )\\\\/\\\\\n             $tongue ||--WWW |\n                ||     ||\nEOC\n";

function converToCliOptions(browserOptions) {
  const cliOptions = {
    e: browserOptions.eyes || 'oo',
    T: browserOptions.tongue || '  ',
    n: browserOptions.wrap,
    W: browserOptions.wrapLength || 40,
    text: browserOptions.text || '',
    _: browserOptions.text || [],
    f: browserOptions.cow,
  };
  if (browserOptions.mode) {
    // converts mode: 'b' to b: true
    cliOptions[browserOptions.mode] = true;
  }
  return cliOptions;
}

function doIt (options, sayAloud) {
  const cow = options.f || DEFAULT_COW;
	const face = faces(options);
	face.thoughts = sayAloud ? "\\" : "o";

	const action = sayAloud ? "say" : "think";
	return balloon[action](options.text || options._.join(" "), options.n ? null : options.W) + "\n" + replacer(cow, face);
}

function say(browserOptions) {
  return doIt(converToCliOptions(browserOptions), true);
}

function think(browserOptions) {
  return doIt(converToCliOptions(browserOptions), false);
}

export { say, think, beavis_zen as BEAVIS_ZEN, bong as BONG, budFrogs as BUD_FROGS, bunny as BUNNY, cheese as CHEESE, cower as COWER, daemon as DAEMON, DEFAULT_COW as DEFAULT, doge as DOGE, dragonAndCow as DRAGON_AND_COW, dragon as DRAGON, elephantInSnake as ELEPHANT_IN_SNAKE, eyes as EYES, flamingSheep as FLAMING_SHEEP, ghostbusters as GHOSTBUSTERS, goat as GOAT, hedgehog as HEDGEHOG, hellokitty as HELLOKITTY, kiss as KISS, kitty as KITTY, koala as KOALA, kosh as KOSH, lukeKoala as LUKE_KOALA, mechAndCow as MECH_AND_COW, meow as MEOW, milk as MILK, moofasa as MOOFASA, moose as MOOSE, mutilated as MUTILATED, ren as REN, satanic as SATANIC, sheep as SHEEP, skeleton as SKELETON, small as SMALL, squirrel as SQUIRREL, stegosaurus as STEGOSAURUS, stimpy as STIMPY, supermilker as SUPERMILKER, surgery as SURGERY, telebears as TELEBEARS, turkey as TURKEY, turtle as TURTLE, tux as TUX, vaderKoala as VADER_KOALA, whale as WHALE, www as WWW };
export { ACKBAR } from '.cows/ackbar.cow';
export { APERTURE_BLANK } from '.cows/aperture-blank.cow';
export { APERTURE } from '.cows/aperture.cow';
export { ARMADILLO } from '.cows/armadillo.cow';
export { ATAT } from '.cows/atat.cow';
export { ATOM } from '.cows/atom.cow';
export { AWESOME_FACE } from '.cows/awesome-face.cow';
export { BANANA } from '.cows/banana.cow';
export { BEARFACE } from '.cows/bearface.cow';
export { BEES } from '.cows/bees.cow';
export { BILL_THE_CAT } from '.cows/bill-the-cat.cow';
export { BIOHAZARD } from '.cows/biohazard.cow';
export { BISHOP } from '.cows/bishop.cow';
export { BLACK_MESA } from '.cows/black-mesa.cow';
export { BOX } from '.cows/box.cow';
export { BROKEN_HEART } from '.cows/broken-heart.cow';
export { C3PO } from '.cows/C3PO.cow';
export { CAKE } from '.cows/cake.cow';
export { CAKE_WITH_CANDLES } from '.cows/cake-with-candles.cow';
export { CAT2 } from '.cows/cat2.cow';
export { CAT } from '.cows/cat.cow';
export { CATFENCE } from '.cows/catfence.cow';
export { CHARIZARDVICE } from '.cows/charizardvice.cow';
export { CHARLIE } from '.cows/charlie.cow';
export { CHESSMEN } from '.cows/chessmen.cow';
export { CHITO } from '.cows/chito.cow';
export { CLAW_ARM } from '.cows/claw-arm.cow';
export { CLIPPY } from '.cows/clippy.cow';
export { COMPANION_CUBE } from '.cows/companion-cube.cow';
export { COWFEE } from '.cows/cowfee.cow';
export { CTHULHU_MINI } from '.cows/cthulhu-mini.cow';
export { CUBE } from '.cows/cube.cow';
export { DALEK } from '.cows/dalek.cow';
export { DALEK_SHOOTING } from '.cows/dalek-shooting.cow';
export { DOCKER_WHALE } from '.cows/docker-whale.cow';
export { DOLPHIN } from '.cows/dolphin.cow';
export { EBI_FURAI } from '.cows/ebi_furai.cow';
export { ELEPHANT2 } from '.cows/elephant2.cow';
export { ELEPHANT } from '.cows/elephant.cow';
export { EXPLOSION } from '.cows/explosion.cow';
export { FAT_BANANA } from '.cows/fat-banana.cow';
export { FAT_COW } from '.cows/fat-cow.cow';
export { FENCE } from '.cows/fence.cow';
export { FIRE } from '.cows/fire.cow';
export { FOX } from '.cows/fox.cow';
export { GHOST } from '.cows/ghost.cow';
export { GLADOS } from '.cows/glados.cow';
export { GOAT2 } from '.cows/goat2.cow';
export { GOLDEN_EAGLE } from '.cows/golden-eagle.cow';
export { HAND } from '.cows/hand.cow';
export { HAPPY_WHALE } from '.cows/happy-whale.cow';
export { HIPPIE } from '.cows/hippie.cow';
export { HIYA } from '.cows/hiya.cow';
export { HIYOKO } from '.cows/hiyoko.cow';
export { HOMER } from '.cows/homer.cow';
export { HYPNO } from '.cows/hypno.cow';
export { IBM } from '.cows/ibm.cow';
export { IWASHI } from '.cows/iwashi.cow';
export { JELLYFISH } from '.cows/jellyfish.cow';
export { KARL_MARX } from '.cows/karl_marx.cow';
export { KILROY } from '.cows/kilroy.cow';
export { KING } from '.cows/king.cow';
export { KITTEN } from '.cows/kitten.cow';
export { KNIGHT } from '.cows/knight.cow';
export { LAMB2 } from '.cows/lamb2.cow';
export { LAMB } from '.cows/lamb.cow';
export { LIGHTBULB } from '.cows/lightbulb.cow';
export { LOBSTER } from '.cows/lobster.cow';
export { LOLLERSKATES } from '.cows/lollerskates.cow';
export { MAILCHIMP } from '.cows/mailchimp.cow';
export { MAZE_RUNNER } from '.cows/maze-runner.cow';
export { MINOTAUR } from '.cows/minotaur.cow';
export { MONA_LISA } from '.cows/mona-lisa.cow';
export { MOOGHIDJIRAH } from '.cows/mooghidjirah.cow';
export { MOOJIRA } from '.cows/moojira.cow';
export { MULE } from '.cows/mule.cow';
export { NYAN } from '.cows/nyan.cow';
export { OCTOPUS } from '.cows/octopus.cow';
export { OKAZU } from '.cows/okazu.cow';
export { OWL } from '.cows/owl.cow';
export { PAWN } from '.cows/pawn.cow';
export { PERIODIC_TABLE } from '.cows/periodic-table.cow';
export { PERSONALITY_SPHERE } from '.cows/personality-sphere.cow';
export { PINBALL_MACHINE } from '.cows/pinball-machine.cow';
export { PSYCHIATRICHELP2 } from '.cows/psychiatrichelp2.cow';
export { PSYCHIATRICHELP } from '.cows/psychiatrichelp.cow';
export { PTERODACTYL } from '.cows/pterodactyl.cow';
export { QUEEN } from '.cows/queen.cow';
export { R2_D2 } from '.cows/R2-D2.cow';
export { RADIO } from '.cows/radio.cow';
export { RENGE } from '.cows/renge.cow';
export { ROBOT } from '.cows/robot.cow';
export { ROBOTFINDSKITTEN } from '.cows/robotfindskitten.cow';
export { ROFLCOPTER } from '.cows/roflcopter.cow';
export { ROOK } from '.cows/rook.cow';
export { SACHIKO } from '.cows/sachiko.cow';
export { SEAHORSE_BIG } from '.cows/seahorse-big.cow';
export { SEAHORSE } from '.cows/seahorse.cow';
export { SHIKATO } from '.cows/shikato.cow';
export { SHRUG } from '.cows/shrug.cow';
export { SMILING_OCTOPUS } from '.cows/smiling-octopus.cow';
export { SNOOPY } from '.cows/snoopy.cow';
export { SNOOPYHOUSE } from '.cows/snoopyhouse.cow';
export { SNOOPYSLEEP } from '.cows/snoopysleep.cow';
export { SPIDERCOW } from '.cows/spidercow.cow';
export { SQUID } from '.cows/squid.cow';
export { SUDOWOODO } from '.cows/sudowoodo.cow';
export { TABLEFLIP } from '.cows/tableflip.cow';
export { TAXI } from '.cows/taxi.cow';
export { TEMPLATE } from '.cows/template.cow';
export { THREADER } from '.cows/threader.cow';
export { THREECUBES } from '.cows/threecubes.cow';
export { TOASTER } from '.cows/toaster.cow';
export { TORTOISE } from '.cows/tortoise.cow';
export { TUX_BIG } from '.cows/tux-big.cow';
export { TWEETY_BIRD } from '.cows/tweety-bird.cow';
export { USA } from '.cows/USA.cow';
export { VADER } from '.cows/vader.cow';
export { WEEPING_ANGEL } from '.cows/weeping-angel.cow';
export { WIZARD } from '.cows/wizard.cow';
export { WOOD } from '.cows/wood.cow';
export { WORLD } from '.cows/world.cow';
export { YASUNA_01 } from '.cows/yasuna_01.cow';
export { YASUNA_02 } from '.cows/yasuna_02.cow';
export { YASUNA_03A } from '.cows/yasuna_03a.cow';
export { YASUNA_03 } from '.cows/yasuna_03.cow';
export { YASUNA_04 } from '.cows/yasuna_04.cow';
export { YASUNA_05 } from '.cows/yasuna_05.cow';
export { YASUNA_06 } from '.cows/yasuna_06.cow';
export { YASUNA_07 } from '.cows/yasuna_07.cow';
export { YASUNA_08 } from '.cows/yasuna_08.cow';
export { YASUNA_09 } from '.cows/yasuna_09.cow';
export { YASUNA_10 } from '.cows/yasuna_10.cow';
export { YASUNA_11 } from '.cows/yasuna_11.cow';
export { YASUNA_12 } from '.cows/yasuna_12.cow';
export { YASUNA_13 } from '.cows/yasuna_13.cow';
export { YASUNA_14 } from '.cows/yasuna_14.cow';
export { YASUNA_16 } from '.cows/yasuna_16.cow';
export { YASUNA_17 } from '.cows/yasuna_17.cow';
export { YASUNA_18 } from '.cows/yasuna_18.cow';
export { YASUNA_19 } from '.cows/yasuna_19.cow';
export { YASUNA_20 } from '.cows/yasuna_20.cow';
export { YMD_UDON } from '.cows/ymd_udon.cow';
export { ZEN_NOH_MILK } from '.cows/zen-noh-milk.cow';
